### ISSUE TITLE
`vhs` is no longer installable on Darwin systems, but isn't marked as such.

### ISSUE THREAD

**Issue by duckpuppy** [NixOS/nixpkgs#220367]
### Describe the bug
f1e06fa1101d18071555410abd2af8e24c929433 introduced `chromium` as a dependency of the `vhs` package. `chromium` is not available on the Darwin platform so `vhs` fails to install with an error stating that `chromium` isn't packaged for Darwin.

### Steps To Reproduce
1. `nix-shell -p vhs` on a darwin system

### Expected behavior
One of two things:
1. `vhs` installs, but does not depend on `chromium` if the system is `*-darwin`
2. `vhs` is marked as being only available on `*-linux` platforms.

### Additional context
```shell
$ nix-shell -p vhs
error: Package ‘chromium-110.0.5481.177’ in /nix/store/daijclw5chp5rqbhwni6xbf9s97h6g66-nixpkgs/nixpkgs/pkgs/applications/networking/browsers/chromium/browser.nix:76 is not supported on ‘x86_64-darwin’, refusing to evaluate.
```

### Notify maintainers
@maaslalani @penguwin 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-darwin"`
 - host os: `Darwin 22.3.0, macOS 10.16`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.12.0`
 - channels(paikens): `"darwin, home-manager, nixpkgs, nur-duckpuppy, nur-gerschtli"`
 - nixpkgs: `/Users/paikens/.nix-defexpr/channels/nixpkgs`
```

**Comment by slinlee** [comment]
@duckpuppy - quick question first, before, when you used `vhs` on darwin, installed via nixpkgs, when you ran it, would it successfully install and run chromium itself? I don't have a darwin system with nix installed to test, so I'm curious.

**Comment by duckpuppy** [comment]
Honestly, I have only recently added vhs to my config because I had a use
for it coming up. I haven't done anything other than run it to see the help
message so I don't know for sure if it ever truly worked. I hadn't got
around to doing a test recording before updating nixpkgs to a version with
the error.

**Comment by penguwin** [comment]
This should be resolved by #222568
